wrappers/nodejs/src/addon.cpp

```diff
@@ -4703,6 +4703,7 @@ void InitModule(v8::Local<v8::Object> exports) {
   _FORCE_SET_ENUM(RS2_FORMAT_Z16H);
   _FORCE_SET_ENUM(RS2_FORMAT_FG);
   _FORCE_SET_ENUM(RS2_FORMAT_Y411);
+  _FORCE_SET_ENUM(RS2_FORMAT_Y16I);
```

Comment:
Can you also upadate C# and Java? (I believe for python it is automatically generated)

Reply:
Done